rosdep/python.yaml

```diff
@@ -5329,6 +5329,9 @@ python-webrtcvad-pip:
     pip:
       packages: [webrtcvad]
 python-websocket:
+  arch:
+    pip:
+      packages: [websocket-client]
```

Comment:
So we already have the [python3-websocket](https://github.com/ros/rosdistro/blob/67cea41a94e36ab0b43c85cddf15a12a7681bb67/rosdep/python.yaml#L7456-L7462) key.  Since we generally prefer the system packages, I think it would be much better to add the arch entry (`python-websocket-client`) there.